AggregateAppEngineUpdater/src/test/java/org/opendatakit/appengine/updater/HostTest.java

```diff
@@ -0,0 +1,73 @@
+package org.opendatakit.appengine.updater;/*
```

Comment:
😭 I hate to be nit-picky but I think this package declaration should go after the header!

Reply:
I was trying to make a surgical change, but I guess you're right :)

Reply:
Thanks for spotting that! There was some problem while copy&pasting